README.md

```diff
@@ -164,6 +164,18 @@ localStorage.debug = "fluence:*";
 In Chromium-based web browsers (e.g. Brave, Chrome, and Electron), the JavaScript console will be default—only to show
 messages logged by debug if the "Verbose" log level is enabled.
 
+## Low level usage
+
+JS client also has an API for low level interaction with AVM and Marine JS.
+It could be handy in advanced scenarios when a user fetches Air dynamically or generates Air without default Aqua compiler.
+
+`callAquaFunction` Allows to call aqua function without schema.
+
+`registerService` Gives an ability to register service without schema. Passed `service` could be
+
+- Plain object. In this case all function properties will be registered as Air service functions.
+- Class instance. All class functions without inherited ones will be registered as Air service functions.
```

Comment:
why not allow to call anything with this low level API including inherited class methods. I would also suggest to call them class methods instead of class functions

Reply:
Agree with renaming.

The decision to forbid inherited props made to make an attempt for avoiding accidental calls of Object methods or Array methods or other prototype chain methods. It could be surprising and forces you to control every method in prototype chain to have robust code(Very hard or impossible if you use external lib).